ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -180,7 +163,7 @@ public OptionalInt valueColumnIndex(final ColumnName fullColumnName) {
   @VisibleForTesting
   public OptionalInt valueColumnIndex(final String fullColumnName) {
```

Comment:
🤔 does this still make sense? can we safely have this return the true index instead of the index within the values? I'm thinking about this in the context of #3417

Reply:
I think that will break all use of the method.

TBH, this probably doesn't belong here.  Nor do the 'copy rowkey & rowtime into the value'. Which could not be in that class with the 'orrible name.

But outside the scope of this PR.